model.py

```diff
@@ -142,74 +147,78 @@ def save(filename):
 
     open(filename, 'w').write(json.dumps(out))
 
+
 def load(filename):
     inc = json.loads(open(filename, 'r').read())
 
     for i, data in enumerate(inc):
         keys[i] = StoredFrame(data)
 
+
 def getparentsticks(stickname):
-    if stickname in ['RIGHT SHOULDER','LEFT SHOULDER','NECK','TORSO']:
+    if stickname in ['RIGHT SHOULDER', 'LEFT SHOULDER', 'NECK', 'TORSO']:
         return []
     if stickname in ['HEAD']:
```

Comment:
Not related to your change but could you change this check to `==` for consistency?

Reply:
Done.

Reply:
Does this change  to be done just in `stickname in ['HEAD']` or with `if stickname in ['RIGHT SHOULDER', 'LEFT SHOULDER', 'NECK', 'TORSO']` too making 4 different if conditions for all

Reply:
Just `[HEAD]` as it's just one item and to be consistent as in other lines when it's just one item.